Insert a check for the musl-fts library which will be used
for *-musl architectures.

--- SConstruct	2015-05-10 01:34:58.000000000 +0200
+++ SConstruct	2015-08-31 11:32:23.251480056 +0200
@@ -286,6 +286,19 @@
     context.Result(rc)
     return rc
 
+def check_musl_fts(context):
+    rc = 1
+    if tests.CheckHeader(context, 'fts.h'):
+        rc = 0
+
+    if tests.CheckLib(context, ['fts']):
+        rc = 0
+
+    conf.env['HAVE_FTS_H'] = rc
+    context.did_show_result = True
+    context.Result(rc)
+    return rc
+
 
 def create_uninstall_target(env, path):
     env.Command("uninstall-" + path, path, [
@@ -448,7 +461,8 @@
     'check_bigfiles': check_bigfiles,
     'check_c11': check_c11,
     'check_gettext': check_gettext,
-    'check_sqlite3': check_sqlite3
+    'check_sqlite3': check_sqlite3,
+    'check_musl_fts': check_musl_fts
 })
 
 if not conf.CheckCC():
